src/backend/src/backend_api.h

```diff
@@ -10,12 +10,18 @@ extern "C" {
 #define DLLExport __attribute__((visibility("default")))
 #endif
 
-DLLExport void runBackend(
+DLLExport struct MavsdkBackend* runBackend(
     const char* system_address,
     const int mavsdk_server_port,
     void (*onServerStarted)(void*),
     void* context);
 
+DLLExport int getPort(struct MavsdkBackend* backend);
```

Comment:
I would prefer `get_port` especially since it's C, and in the other calls as well.

Reply:
You're right. Back when I started I followed gRPC's naming, but that was a mistake. I'll change that, either in this PR or in a following one that would also fix the "dc" and "backend" everywhere.